exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -114,6 +125,105 @@ pub struct Run {
     jvm_args_append: Vec<String>,
 }
 
+/// EJB-specific `run-dev` command.
+///
+/// Automatically generates node configuration for one
+/// validator and runs it using provided `artifacts_path` as a directory for Java service artifacts.
+#[derive(Debug, StructOpt, Serialize, Deserialize)]
+#[structopt(rename_all = "kebab-case")]
+pub struct RunDev {
+    /// Path to the directory containing Java service artifacts.
+    #[structopt(long)]
+    artifacts_path: PathBuf,
+    /// Path to a database directory. If not provided, system temporary directory is used.
+    ///
+    /// If not provided, new database will be created for each use of the command. Database is
+    /// not deleted automatically, the user must clean up database directory themselves.
+    #[structopt(long, short = "d")]
+    db_path: Option<PathBuf>,
+    /// Path to a node configuration file. If not provided, autogenerated node configuration is
+    /// used.
+    ///
+    /// If not provided, new node configuration will be generated for each use of the command.
+    /// Defaults to <db_path>/config if <db_path> parameter is provided. Otherwise system temporary
+    /// directory is used. Config files are not deleted automatically, the user must clean up
+    /// configuration files themselves.
+    #[structopt(long, short = "c")]
+    node_config: Option<PathBuf>,
+    /// Path to log4j configuration file.
+    #[structopt(long)]
+    ejb_log_config_path: Option<PathBuf>,
+}
+
+impl RunDev {
+    /// Returns path to a directory for node configuration files.
+    ///
+    /// If <db_path> parameter is provided, returns <db_path>/config. Else returns system temporary
```

Comment:
It does not return 'system temporary directory', it creates a new temporary directory.